engine/lib/views.php

```diff
@@ -1367,18 +1367,36 @@ function elgg_view_list_item($item, array $vars = array()) {
  *
  * @param string $name  The specific icon to display
  * @param string $class Additional class: float, float-alt, or custom class
+ * @param array  $vars  Variables to pass to the view.
  *
  * @return string The html for displaying an icon
  */
-function elgg_view_icon($name, $class = '') {
+function elgg_view_icon($name, $class = '', $vars = array()) {
```

Comment:
same as done for getIconURL()? mixed param? fine by me, but can't type hint then anymore :)